developer/wip/for-adopters/Getting Started.md

```diff
@@ -0,0 +1,123 @@
+complete demonstration dataspace between two organizations. The MVD includes automated setup of a complete dataspace
+environment in a few minutes.
+
+## Overview: Key Components
+
+EDC is architected as modules called ***extensions*** that can be combined and customized to create ***components***
+that perform specific tasks. These components (the "C" in EDC) are **not** what is commonly referred to as "
+microservices." Rather, EDC components may be deployed as separate services or collocated in a runtime process. This
+section provides a quick overview of the key EDC components.
+
+### The Connector
+
+The Connector is a pair of components that *control* data sharing and *execute* data transfer. These components are the
+**_Control Plane_** and **_Data Plane,_** respectively. In keeping with EDC's modular design philosophy, connector
+components may be deployed in a single monolith (for simple use cases) or provisioned as clusters of individual
+services. It is recommended to separate the Control Plane and Data Plane so they can be individually managed and scaled.
+
+![[Control and Data Planes.svg]]
+
+#### The Control Plane
+
+The Control Plane is responsible for creating contract agreements that grant access to data, managing data transfers,
+and monitoring usage policy compliance. For example, a data consumer's control Plan initiates a contract negotiation
+with a data provider's connector. The negotiation is an asynchronous process that results in a *contract agreement* if
+approved. The consumer connector then uses the contract agreement to initiate a *data transfer* with the provider
+connector. A data transfer can be a one-shot (finite) transfer, such as a discrete set of data, or an ongoing (
+non-finite) data stream. The provider control plane can pause, resume, or terminate transfers in response to certain
+conditions. For example, if a contract agreement expires.
+
+#### The Data Plane
+
+The Data Plane is responsible for executing data transfers, which are managed by the Control Plane. A Data Plane sends
+data using specialized technology such as a messaging system or data integration platform. EDC includes the *Data Plane
+Framework (DPF)* for building custom Data Planes. Alternatively, a Data Plane can be built using other languages or
+technologies and integrated with the EDC Control Plane by implementing
+the [Data Plane Signaling API](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling.md).
+
+### Federated Catalog
+
+The Federated Catalog (FC) is responsible for crawling and caching data catalogs from other participants. The FC builds
+a local cache that can be queried or processed without resorting to complex distributed queries across multiple
+participants.
+
+### Identity Hub
+
+The Identity Hub securely stores and manages W3C Verifiable Credentials, including the presentation of VCs and the
+issuance and re-issuance process.
```

Comment:
maybe worth mentioning that it is the reference implementation of DCP

Reply:
Let's do this in a top section later with DSP as well